e2e-tests/micronaut-rest-api/src/test/java/com/foo/micronaut/rest/MicronautRestApiTest.java

```diff
@@ -0,0 +1,21 @@
+package com.foo.micronaut.rest;
+
+import io.micronaut.runtime.EmbeddedApplication;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Assertions;
+
+import jakarta.inject.Inject;
+
+@Disabled
+class MicronautRestApiTest {
```

Comment:
the test is disabled also it fails as checked. if you want to keep it, add some comments, otherwise, might remove it.

Reply:
Removed it